util/suffix.c

```diff
@@ -69,6 +69,47 @@ const char *suffix_si_get(double *value)
 	return "";
 }
 
+static struct si_suffix_ld {
+	long double magnitude;
+	const char *suffix;
+} si_suffixes_ld[] = {
+	{1e30, "Q"},
+	{1e27, "R"},
+	{1e24, "Y"},
+	{1e18, "E"},
+	{1e15, "P"},
+	{1e12, "T"},
+	{1e9, "G"},
+	{1e6, "M"},
+	{1e3, "k"},
+	{1e0, ""},
+	{1e-3, "m"},
+	{1e-6, "u"},
+	{1e-9, "n"},
+	{1e-12, "p"},
+	{1e-15, "f"},
+	{1e-18, "a"},
+	{1e-21, "z"},
+	{1e-24, "y"},
+	{1e-27, "r"},
+	{1e-30, "q"},
```

Comment:
You can drop the unit types for `< 0`.  It's not relevant for nvme. And instead duplicating these constants from just above, make these reusable.